src/utils/workspace.ts

```diff
@@ -0,0 +1,51 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { IAzureQuickPickItem, IAzureUserInput } from 'vscode-azureextensionui';
+
+export async function selectWorkspaceFile(ui: IAzureUserInput, placeHolder: string, getSubPath?: (f: vscode.WorkspaceFolder) => string | undefined): Promise<string> {
```

Comment:
This repo already has some workspace functions here: https://github.com/Microsoft/vscode-azureappservice/blob/master/src/util.ts#L41

Can we merge these?